pkgs/development/rocm-modules/5/hsa-amd-aqlprofile-bin/default.nix

```diff
@@ -29,11 +25,13 @@ in stdenv.mkDerivation (finalAttrs: {
     runHook preInstall
 
     mkdir -p $out
-    cp -a opt/rocm-${version}/* $out
+    cp -a opt/rocm-${finalAttrs.version}/* $out
 
     runHook postInstall
   '';
 
+  passthru.updateScript = updateScript { inherit (finalAttrs) version; };
```

Comment:
```suggestion
  passthru.updateScript = (callPackage ./update.nix { }) { inherit (finalAttrs) version; };
```

Reply:
Weird. Tried that before and it didn't work, but does now.